core/trino-main/src/main/java/io/trino/server/tracing/WorkerDynamicTracingResource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.server.tracing;
+
+import com.google.inject.Inject;
+import io.trino.server.security.ResourceSecurity;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.Response;
+
+import static io.trino.server.security.ResourceSecurity.AccessType.INTERNAL_ONLY;
+import static java.util.Objects.requireNonNull;
+
+@Path("/v1/tracing")
```

Comment:
do we need an endpoint for that? Wouldn't JMX be not enough?

Reply:
@losipiuk endpoint is easier to automate than JMX

Reply:
You could say the same for logging. Yet we do not have endpoint for that.

Reply:
We have a config for logging and logging has a static behavior

Reply:
Harder to integrate. You need to use java to coordinate

Reply:
probably true - I just do not like the inconsistency. Maybe we should have http endpoint for logging too. This is quite common we want to change log level at runtime